src/EnergyPlus/CurveManager.cc

```diff
@@ -3019,19 +3107,93 @@ namespace Curve {
     }
 
     void GetCurveMinMaxValues(EnergyPlusData &state,
-                              int const CurveIndex,                // index of curve in curve array
-                              Real64 &Var1Min,                     // Minimum values of 1st independent variable
-                              Real64 &Var1Max,                     // Maximum values of 1st independent variable
-                              ObjexxFCL::Optional<Real64> Var2Min, // Minimum values of 2nd independent variable
-                              ObjexxFCL::Optional<Real64> Var2Max, // Maximum values of 2nd independent variable
-                              ObjexxFCL::Optional<Real64> Var3Min, // Minimum values of 3rd independent variable
-                              ObjexxFCL::Optional<Real64> Var3Max, // Maximum values of 3rd independent variable
-                              ObjexxFCL::Optional<Real64> Var4Min, // Minimum values of 4th independent variable
-                              ObjexxFCL::Optional<Real64> Var4Max, // Maximum values of 4th independent variable
-                              ObjexxFCL::Optional<Real64> Var5Min, // Minimum values of 5th independent variable
-                              ObjexxFCL::Optional<Real64> Var5Max, // Maximum values of 5th independent variable
-                              ObjexxFCL::Optional<Real64> Var6Min, // Minimum values of 6th independent variable
-                              ObjexxFCL::Optional<Real64> Var6Max  // Maximum values of 6th independent variable
+                              int const CurveIndex, // index of curve in curve array
```

Comment:
Yeah, this worker function isn't doing any error checking or anything, so when we clean the API, we probably just don't even need this extra layer.  The client can simply grab the members off of the curve instance itself.